docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-update.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: update
+permalink: reference/mesheryctl/system/update
+type: reference
+display-title: "false"
+language: en
+lang: en
+categories: en
+list: exclude
+# image: /assets/img/platforms/brew.png
+---
+
+<!-- Copy this template to create individual doc pages for each mesheryctl commands -->
+
+<!-- Name of the command -->
+# mesheryctl system update
+
```

Comment:
I would be great to have a short description of the purpose of the command and a link to the Upgrade Guide - https://docs.meshery.io/guides/upgrade

Reply:
Could you suggest some description?

Reply:
Refer this command tracker spreadsheet... (assuming you have access to the drive) https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit?usp=sharing

Reply:
But the description part is being fixed on PR #2985 

Ok here's a debrief. `mesheryctl system update` just updates the container images and not the `mesheryctl` itself. So you need to add addl. description that "if you want to update `mesheryctl` then refer this guide" like that you can come up and add it to the page

Reply:
I don't but I have requested for the permission.

Reply:
It's okay. Someone from the org will look at it and accept for viewing